Mathlib/SetTheory/Nimber/Field.lean

```diff
@@ -0,0 +1,231 @@
+/-
+Copyright (c) 2024 Violeta Hernández Palacios. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Violeta Hernández Palacios
+-/
+import Mathlib.Algebra.CharP.Two
+import Mathlib.SetTheory.Nimber.Basic
+import Mathlib.Tactic.Abel
+
+/-!
+# Nimber multiplication and division
+
+The nim product `a * b` is recursively defined as the least nimber not equal to any
+`a' * b + a * b' + a' * b'` for `a' < a` and `b' < b`. When endowed with this operation, the nimbers
+form a field.
+
+## Todo
```

Comment:
Are there any interesting interactions between the resulting ring / field and the order structure? If so, can you add them to this list?

Reply:
As far as I'm aware, the ordering on nimbers doesn't interact with the field structure in any "simple" way. (There's stuff like the simplest extension theorems, but ironically those are far from simple and are probably best explained when we actually get to proving them).

Reply:
So it's not an ordered ring or similar?

Reply:
It's a characteristic 2 field, which is probably the furthest you can get from an ordered ring :P